doc/releases/changelog-dev.md

```diff
@@ -255,6 +255,9 @@
   `jnp.arange`, and `jnp.full`.
   [#6865)](https://github.com/PennyLaneAI/pennylane/pull/6865)
 
+* The `QROM` template is upgraded to decompose more efficiently when `work_wires` are not used.
+  [#6967)](https://github.com/PennyLaneAI/pennylane/pull/6967)
+
```

Comment:
This should be in the bug fix section not the Capturing and representing hybrid programs